k9mail/src/main/java/com/fsck/k9/K9.java

```diff
@@ -584,29 +585,45 @@ private void updateUnreadWidget() {
                 }
             }
 
+            private void updateMailListWidget() {
+                try {
+                    MessageListWidgetProvider.updateMailViewList(K9.this);
+                } catch (Exception e) {
```

Comment:
This only catches RuntimeExceptions, right? Might as well annotate that (I was confused about the `throw e` at first because the method doesn't have any checked exceptions)